core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1924,6 +1924,22 @@ public static int truncate(int v, int x) {
     return v - remainder;
   }
 
+  /** SQL {@code LAST_DAY} function. */
+  public static int lastDay(int date) {
```

Comment:
Please add javadoc for `date` argument. Is it days since epoch ?

Reply:
Ok.